geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/tests/VMDistributedTest.java

```diff
@@ -89,7 +89,7 @@ public void testReturnValue() throws Exception {
     final Host host = Host.getHost(0);
     final VM vm = host.getVM(0);
     // Assert class static invocation works
-    AsyncInvocation a1 = vm.invokeAsync(VMDistributedTest::getAndIncStaticCount);
+    AsyncInvocation<?> a1 = vm.invokeAsync(VMDistributedTest::getAndIncStaticCount);
```

Comment:
This class is kind of a mess what with all the deprecated calls and use of JUnit assertions rather than AssertJ, but this would probably be better as:
```

    AsyncInvocation<Integer> a1 = vm.invokeAsync(VMDistributedTest::getAndIncStaticCount);
    a1.join();
    assertEquals(0, (long) a1.getReturnValue());
    // Assert class static invocation with args works
    a1 = vm.invokeAsync(() -> incrementStaticCount(2));
    a1.join();
    assertEquals(3, (long) a1.getReturnValue());
    // Assert that previous values are not returned when invoking method w/ no return val
    a1 = vm.invokeAsync(VMDistributedTest::incStaticCount);
    a1.join();
    assertNull(a1.getReturnValue());
    // Assert that previous null returns are over-written
    a1 = vm.invokeAsync(VMDistributedTest::getAndIncStaticCount);
    a1.join();
    assertEquals(4, (long) a1.getReturnValue());

    // Assert object method invocation works with zero arg method
    final VMTestObject o = new VMTestObject(0);
    a1 = vm.invokeAsync(o, "incrementAndGet", new Object[] {});
    a1.join();
    assertEquals(1, (long) a1.getReturnValue());
```
so that we can use a definite type rather than `<?>`

Reply:
Good idea. Doing this now.

Reply:
Thanks for the quick turnaround @yozaner1324 👍